client/i18n/locales/english/intro.json

```
@@ -2207,6 +2207,12 @@
           "In these lessons, you will learn when you should use certain semantic elements like the <code>em</code> element over the <code>i</code> element, description lists, and more."
         ]
       },
+      "workshop-major-browsers-list": {
+        "title": "Build a List of Major Web Browsers",
+        "intro": [
+          "In this lesson you are going to get familiar with some major browsers and you will also learn how to use semantic tags like `dl`, `dt`, and `dd` visually."
```

```suggestion
          "In this workshop you are going to practice using semantic tags like `dl`, `dt`, and `dd`."
```

this is not a lesson